tests/features/search_and_filters/search_and_filters.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+test.describe('Check filters and search', () => {
+  test.describe('Check filters', () => {
+      test.beforeEach(async ({steps: {pages}, page}) => {
+          await test.step(`I open catalog page`, async () => {
+              await page.goto('');
+              await pages.topPanel.goToCatalog();
```

Comment:
I propose to rename `goToCatalog()` into `clickOnCatalog` or something like this. As the current name could be considered as navigating to some URL. 

In this method we just click on the button